CHANGELOG-0.2.md

```diff
@@ -1,5 +1,20 @@
 # Changelog 0.2
 
+## [0.2.2] 2019-03-27
+
+### Added
+
+- Documentation for manual upgrade Kubernetes from 1.13.0 to 1.13.1 [link](./docs/home/HOWTO.md#how-to-upgrade-kubernetes-cluster-from-1.13.0-to-1.13.1)
```

Comment:
Please correct link (without dots): #how-to-upgrade-kubernetes-cluster-from-1130-to-1131